src/extensions/common/PlotGrid.tsx

```diff
@@ -2,6 +2,7 @@ import { Button, Grid } from '@material-ui/core';
 import React, { FunctionComponent, useState } from 'react';
 import { CalculationPool, HitherContext, Sorting } from '../extensionInterface';
 import ClientSidePlot from './ClientSidePlot';
+import './localStyles.css';
```

Comment:
I needed to move localStyles.css so that it would apply the jupyter widgets as well as the web gui.

Reply:
Makes sense.